content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -117,9 +117,9 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
 
 {{< feature-state state="beta" for-kubernetes-version="1.12" >}}
 
-A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the
+A DaemonSet resource type ensures that all eligible nodes run a copy of a Pod. Normally, the
 node that a Pod runs on is selected by the Kubernetes scheduler. However,
-DaemonSet pods are created and scheduled by the DaemonSet controller instead.
+DaemonSet pods can also be created and scheduled by a DaemonSet controller instead.
 That introduces the following issues:
```

Comment:
This text reads like it's describing a new feature, rather than something that has been the default for several releases.
`ScheduleDaemonSetPods` is the default behavior and, even though it's in beta, to me it makes sense to document the older behavior as the exceptional one.

@christophercarney do you agree, and if so do you want to cover that clean-up in this PR?

Reply:
I agree and that was something that occurred to me as I was writing so I'm glad you pointed that out. I will make that change in the scope of this PR.